webapp/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,51 @@
+import { ReactComponent as IconArrowBack } from './icons/arrow-back.svg';
+import { ReactComponent as IconArrowForward } from './icons/arrow-forward.svg';
+import { ReactComponent as IconChat } from './icons/chat.svg';
+import { ReactComponent as IconEdit } from './icons/edit.svg';
+import { ReactComponent as IconFile } from './icons/file.svg';
+import { ReactComponent as IconLogin } from './icons/login.svg';
+import { ReactComponent as IconLogout } from './icons/logout.svg';
+import { ReactComponent as IconPlay } from './icons/play.svg';
+import { ReactComponent as IconSearch } from './icons/search.svg';
+import { ReactComponent as IconUsers } from './icons/users.svg';
+
```

Comment:
What about the `size`? Each icon seem to have its own size. We should export them with a normalized size like in Figma

Reply:
Also, as said, I think we also have to think about a way to set `colors`

Reply:
Good catch. I've resized the icons to the same size (24px).